benchmarks/cuda/toy_detector_cuda.cpp

```diff
@@ -129,7 +125,7 @@ BENCHMARK_DEFINE_F(ToyDetectorBenchmark, CUDA)(benchmark::State& state) {
                                params_cuda_buffer);
 
             // Run track fitting
-            traccc::track_state_container_types::buffer
+            traccc::edm::track_fit_container<traccc::default_algebra>::buffer
```

Comment:
More future-proof to use the algorithm's output type here.

Reply:
The applications are a bit all over the place with their formalism. In this PR I was going to keep the current idioms everywhere. Not wanting to change more things than necessary.